tardis/io/config_reader.py

```diff
@@ -486,7 +487,9 @@ def parse_spectrum_list2dict(spectrum_list):
     """
     Parse the spectrum list [start, stop, num] to a list
     """
-
+    if isinstance(spectrum_list, dict):
```

Comment:
how about we change to 'if 'start' in spectrum_list and 'stop' in spectrum list and 'num' in spectrum_list`: :wink: